module/solutions/helper.js

```diff
@@ -3174,16 +3174,30 @@ module.exports = class SolutionsHelper {
         if (!solutionDocument.length > 0) {
           throw new Error(messageConstants.apiResponses.SOLUTION_NOT_FOUND);
         }
+        
```

Comment:
@borkarsaish65 we can remove unwanted code block

Reply:
done